synfig-core/src/synfig/valuenode_registry.h

```diff
@@ -35,38 +35,13 @@
 /* === M A C R O S ========================================================= */
 
 /// This macro automatically registers valuenode and defines its
-/// get_name & get_local_name functions.
-/// NOTE: this relies on valuenode being either in synfig namespace
-/// or in 'used' namespace.
-/// TODO: move all nodes into synfig::valuenodes namespace
-#define REGISTER_VALUENODE(klass, _version, _name, _local_name)				\
-	namespace synfig 														\
-	{																		\
-		class Register_##klass:												\
-			public RegisterValueNode<klass, Register_##klass>				\
-		{																	\
-			public:															\
-				static const ReleaseVersion release_version;				\
-				static const char* name;									\
-				static const char* local_name;								\
-		};																	\
-																			\
-		String																\
-		klass::get_name() const												\
-		{																	\
-			return _name;													\
-		}																	\
-																			\
-		String																\
-		klass::get_local_name()const										\
-		{																	\
-			return _local_name;												\
-		}																	\
-																			\
-		const ReleaseVersion Register_##klass::release_version = _version;	\
-		const char* Register_##klass::name = _name;							\
-		const char* Register_##klass::local_name = _local_name;				\
-	}
+/// get_name & get_local_name methods.
+#define REGISTER_VALUENODE(klass, _version, _name, _local_name) \
+static bool result_##klass = synfig::ValueNodeRegistry::register_node_type(_name, \
```

Comment:
Don't you think indentation would help realize next lines are part of `#define`?

Reply:
How do you propose to indent them?

Reply:
@rodolforg 
Done.